Gallery-Server/src/main/java/edu/yu/cs/gallery/GalleryResource.java

```diff
@@ -107,12 +114,12 @@ public Response deleteById(@PathParam("id") Long id, @Context UriInfo uriInfo) t
     @POST
     @Transactional
     public Response create(Gallery gallery, @Context UriInfo uriInfo) throws UnknownHostException, MalformedURLException {
-        if (this.gallery != null) {
-            return Response.status(502, "This Gallery-Server already has a gallery assigned").build();
+        if (utility.gallery != null) {
+            return Response.status(409, "utility Gallery-Server already has a gallery assigned").build();
```

Comment:
the error message is a little wonky, the user has no idea what `utility` means

Reply:
Ok